### PR TITLE
Update entities for db models and add migrations

### PR DESCRIPTION
**Pull request by RasmusBroborg**
#65.

### CODE REVIEW

File: src/GT.Data/Data/GTAppDb/Entities/ListingInquiry.cs [hunk 1]
```diff
@@ -9,10 +8,10 @@ public class ListingInquiry : IGTEntity
...
 
 		[Column(TypeName = "nvarchar(100)")]
-		public string MessageTitle { get; set; }
+		public string? MessageTitle { get; set; }
 
 		[Column(TypeName = "nvarchar(500)")]
-		public string MessageBody { get; set; }
+		public string? MessageBody { get; set; }
 
 		[Column(TypeName = "nvarchar(100)")]
```
Comment (JohanSaisa) [inline src/GT.Data/Data/GTAppDb/Entities/ListingInquiry.cs:16]: Propose nvarchar(254).

**Review by JohanSaisa — CHANGES_REQUESTED** [review]
Please see comment and Update applicant email to nvarchar(254) in InquiryDTO.

File: src/GT.Data/Data/GTAppDb/Entities/ListingInquiry.cs (hunk 1 above)
Comment (RasmusBroborg) [reply src/GT.Data/Data/GTAppDb/Entities/ListingInquiry.cs:16]: After an internal discussion it was decided that 500 was to be kept until further notice. Resolving comment.

Comment (JohanSaisa) [reply src/GT.Data/Data/GTAppDb/Entities/ListingInquiry.cs:16]: Please re-read.

**Review by JohanSaisa — CHANGES_REQUESTED** [review]
Approved.

**Review by JohanSaisa — APPROVED** [review]
Approved.

**Pull request merged**